options/options.go

```diff
@@ -211,6 +211,9 @@ func (terragruntOptions *TerragruntOptions) SaveVariables() (err error) {
 
 // ImportVariablesFromFile loads variables from the file indicated by path
 func (terragruntOptions *TerragruntOptions) ImportVariablesFromFile(path string, origin VariableSource) error {
+	if !strings.Contains(path, "/") {
```

Comment:
Why checking if the path contains /. Isn't preferable to check if the path is absolute?

Reply:
What we would need is the absolute path from the working dir. The current working dir is not necessarily the terragruntOptions working dir so the filepath.Abs method does not work. Do you know of any method that would resolve the absolute path relative from another working dir than the current one?

Ex: I am in ~/, I am running a terragrunt project in ~/terragrunt/. The file reference is "file.var". filePath.Abs would return ~/file.var, the real absolute path would be ~/terragrunt/file.var.

Adding up the working dir when the file path is not absolute can lead to issues for the same reasons mentionned above.

Reply:
By the way, such function exists. `utils.GetPathRelativeTo` and others in the same family.,

Once you get the relative path, it is easy to convert it to an absolute path.